RestaurantReservation.Db/RestaurantReservationDbContext.cs

```diff
@@ -0,0 +1,205 @@
+﻿using Microsoft.EntityFrameworkCore;
+using RestaurantReservation.Db.Models;
+
+namespace RestaurantReservation.Db
+{
+    public class RestaurantReservationDbContext : DbContext
+    {
+        public DbSet<CustomerDTO> Customers { get; set; }
+        public DbSet<EmployeeDTO> Employees { get; set; }
+        public DbSet<MenuItemDTO> MenuItems { get; set; }
+        public DbSet<OrderDTO> Orders { get; set; }
+        public DbSet<OrderItemDTO> OrderItems { get; set; }
+        public DbSet<RestaurantDTO> Restaurants { get; set; }
+        public DbSet<TableDTO> Tables { get; set; }
+        public DbSet<ReservationDTO> Reservations { get; set; }
+        public DbSet<ReservationsWithCustomerAndRestaurant> ReservationsWithCustomerAndRestaurant { get; set; }
+        public DbSet<EmployeesWithRestaurant> EmployeesWithRestaurant { get; set; }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            optionsBuilder.UseSqlServer("Data Source=(localdb)\\MSSQLLocalDB;" +
+            "Initial Catalog=RestaurantReservationCore;");
+        }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            OnDeleteSetNullForForeignKeys(modelBuilder);
+
+            SeedCustomers(modelBuilder);
```

Comment:
This class is getting huge, I recommend to move out the seed methods to a separate extension class